### PR TITLE
Add git guide for dummies

### PR DESCRIPTION
**Pull request by vyaspranjal33**
added Git Beginner's Guide for Dummies

### CODE REVIEW

File: LearningX.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ This has useful links to learn X.
 4. [GitHub Help Guide](https://help.github.com/)
 5. [IRC](https://fedoramagazine.org/beginners-guide-irc/)
 6. [Git and Github in Plain English](https://red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
+7. [Git Beginner's Guide for Dummies](http://backlogtool.com/git-guide/en)
```
Comment (tapaswenipathak) [inline LearningX.md]: Does this link lead you somewhere? I didn't find any git guide on this website, closing the pull request. Feel free to add another commit I'll merge.

Comment (tapaswenipathak) [reply LearningX.md]: Does this link lead you somewhere? I didn't find any git guide on this website, closing the pull request. Add another commit having the guide, I'll merge.

Comment (tapaswenipathak) [inline LearningX.md]: Does this link lead you somewhere? I didn't find any git guide on this website, closing the pull request. Add another commit having the guide I'll merge.

**Review by tapaswenipathak — COMMENTED** [review]
I didn't find any git guide on this url, closing the pull request.

**Comment by vyaspranjal33** [comment]
please Re open , i've the new url to add  Or would u like to have a new pr

**Comment by tapaswenipathak** [comment]
lol I didn't notice, why do you call it for dummies? The tutorial does not say so.

**Comment by vyaspranjal33** [comment]
> what's in  the name - "Shakespeare"
:smiley:

**Comment by tapaswenipathak** [comment]
If the tutorial's title does not have *for dummies*, it should be changed to *for newbies* or something i.e., put **any** better word.

Updated the url text per the title of the tutorial.
